homeassistant/components/mastodon/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: |
+      Legacy Notify platform can't be created within async-setup.
```

Comment:
IMO the rule doesn't target the notify services

Reply:
OK, I thought the notify services were a bit of a grey area so why I commented, will mark as done.